runtime-modules/storage/src/lib.rs

```diff
@@ -3942,4 +3602,342 @@ impl<T: Trait> Module<T> {
             distribution_bucket_index,
         }
     }
+
+    fn generate_new_storage_buckets(
```

Comment:
The name is confusing - we don't create new buckets here.

Reply:
:+1: